src/content/docs/en/guides/styling.mdx

```diff
@@ -136,14 +136,15 @@ const backgroundColor = "rgb(24 121 78)";
 
 In Astro, HTML attributes like `class` do not automatically pass through to child components.
 
-
 Instead, accept a `class` prop in the child component and apply it to the root element. When destructuring, you must rename it, because `class` is a [reserved word](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#reserved_words) in JavaScript.
 
+You must also pass the `data-astro-cid-*` attribute, unless you have changed `scopedStyleStrategy` to `'class'` or `'where'`. You can do this by passing the `...rest` of the props to the component.
```

Comment:
```suggestion
Using the default scoped style strategy, you must also pass the `data-astro-cid-*` attribute. You can do this by passing the `...rest` of the props to the component. If you have changed `scopedStyleStrategy` to `'class'` or `'where'`, the `...rest` prop is not necessary.
```
Just separating more clearly the two cases here, instead of mixing the "unless" in the middle of the need and the way to achieve it.

(This also means we can remove line 164, I think?)